t/full-stack.t

```diff
@@ -292,17 +292,17 @@ subtest 'Cache tests' => sub {
     ok $cache_client->info->available_workers, 'cache service worker is available';
 
     $job_name = 'tinycore-1-flavor-i386-Build1-core@coolone';
-    client_call('-X POST jobs/3/restart', qr|test_url.+3.+tests.+5|, 'client returned new test_url');
+client_call('-X POST jobs/3/restart', qr|test_url.+3.+tests.+5|, 'client returned new test_url for test 5');
```

Comment:
I don't care much but this line is not messed (by tidy).

Reply:
done